After wrapping gnc-fq-check and gnc-fq-helper we can no longer execute them
with perl, so execute them directly instead.

--- gnucash-2.6.6/src/scm/price-quotes.scm.orig	2014-04-27 17:42:28.000000000 -0500
+++ gnucash-2.6.6/src/scm/price-quotes.scm	2015-07-09 16:12:11.196218472 -0500
@@ -74,7 +74,7 @@
     (define (start-program)
       (if (not (string-null? gnc:*finance-quote-check*))
           (set! program (gnc-spawn-process-async
-                         (list "perl" "-w" gnc:*finance-quote-check*) #t))))
+                         (list gnc:*finance-quote-check*) #t))))
 
     (define (get-sources)
       (if (not (null? program))
@@ -158,7 +158,7 @@
     (define (start-quoter)
       (if (not (string-null? gnc:*finance-quote-helper*))
           (set! quoter (gnc-spawn-process-async
-                        (list "perl" "-w" gnc:*finance-quote-helper*) #t))))
+                        (list gnc:*finance-quote-helper*) #t))))
 
     (define (get-quotes)
       (if (not (null? quoter))
